javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/AgentTracerProviderConfigurer.java

```diff
@@ -44,6 +46,18 @@ public void configure(SdkTracerProviderBuilder sdkTracerProviderBuilder) {
     sdkTracerProviderBuilder.addSpanProcessor(new AddThreadDetailsSpanProcessor());
 
     maybeConfigureExporterJar(sdkTracerProviderBuilder);
+    maybeEnableLoggingExporter(sdkTracerProviderBuilder);
+  }
+
+  private static void maybeEnableLoggingExporter(SdkTracerProviderBuilder builder) {
+    if (Config.get().isAgentDebugEnabled()) {
+      // todo: is this really a reliable way to detect this?
+      // don't install another instance if the user has already explicitly requested it.
+      if (!Config.get().getProperty("otel.traces.exporter", "").equalsIgnoreCase("logging")
```

Comment:
This is hard to read.  First thought was that putting the `Config.get()` into a local var might help...but also, what about factoring out a method called `hasLoggingExporter()` or something similar?

Reply:
done. check out the refactoring

Reply:
Whoopsie....it now reads "if logging is enabled, enable logging".  I think the method might want to be called `loggingExporterIsNotAlreadyConfigured` if you don't wanna invert the logic.

Reply:
fixed